neqo-http3/src/connection.rs

```diff
@@ -579,15 +584,19 @@ impl Http3Connection {
         let mut found = false;
 
         if let Some(request_stream) = &mut self.request_streams_client.get_mut(&stream_id) {
-            qdebug!([label] "Request/response stream {} is readable.", stream_id);
+            qdebug!(
+                [label],
+                "Request/response stream {} is readable.",
```

Comment:
What a shame that the extra character causes rustfmt to reflow this...
```suggestion
                "Request/response stream {} is readable",
```